src/incentives/utils/create-transaction.js

```diff
@@ -281,6 +699,12 @@ const createTransaction = {
         status: httpStatus.NOT_IMPLEMENTED,
         errors: { message: "Not Yet Implemented" },
       };
+      const { device_id } = request.params;
+      /**
+       * We shall find a way of connecting to the Device Registry
```

Comment:
@BenjaminSsempala , we need to start the considerations of developing packages or libraries that will enable people to access this `device-registry` data by just installing them in their apps without having to make API calls. These libs can be made to support all major programming languages